modules/evm-bridge/src/lib.rs

```diff
@@ -74,6 +74,8 @@ pub mod module {
 		ExecutionError,
 		/// Invalid return value
 		InvalidReturnValue,
+		/// Number overflow
+		NumberOverflow,
 	}
```

Comment:
There is a built in `ArithmeticError::Overflow` in `DispatchError`, in `sp_runtime`

https://github.com/paritytech/substrate/blob/3cdb30e1ecbafe8a866317d4550c921b4d686869/primitives/runtime/src/lib.rs#L589

Reply:
@ferrell-code yeah but I wan't sure because we're not doing any arithmetic but just downgrading to u128 so thought this might make more sense. But I see know we're already using it. Updating...